### ISSUE TITLE
Script doesn't inherit node.

### ISSUE THREAD

**Issue by EReeves** [neikeq/GodotSharp#41]
Hi, just built this on the Godot master branch and I can't get it to work. I'm sure it's probably a syntax error or something small I'm overlooking so sorry in advance.

Startup class I'm using to test if I built it right or not.
`
using Godot;
using System;

public class Startup : Node
{
    public override void _Ready(){
        SetProcessInput(true);
    }

    public override void _Input(InputEvent ie) {
        if(Input.IsActionPressed("ui_cancel")) {
            GetTree().Quit();
        }
    }
}
`

The message I get when I run 
ERROR: start: Script does not inherit a Node: res://src/Startup.cs

Help would be appreciated, thanks.

**Comment by NathanWarden** [comment]
Make sure you did the following first:

1) Copy the mono folder into godot/modules/mono
2) Build with "scons p=x11 mono_glue=no"
3) Run "bin\godot.x11.tools.64 --generate-mono-glue modules/mono/glue"
4) Build again with "scons p=x11"

That should get you the proper build.

Next, make sure the script name and the class name match. IE: "class Startup" is in the file "Startup.cs"

Also, you don't need "SetProcessInput(true);" Godot automatically handles this now.

Hope that helps :)

**Comment by neikeq** [comment]
I can't reproduce it. You're probably using a binary built with `mono_glue=no`.

**Comment by neikeq** [comment]
I changed it to print a warning at startup if mono glue is disabled. I also updated the building guide to make this clear. It should be less confusing now.

**Comment by EReeves** [comment]
I don't think that was it, I think the binary wasn't generating the glue properly after the first build. I noticed a bunch of text indicating it was generating after starting from scratch with updated repositories.

Such as
```
Generating World...
Generating World2D...
Generating WorldEnvironment...
Generating XMLParser...
Generating YSort...
```
Which didn't show up the first time.

I'm not sure why, I tried building it a few times before posting the issue, but it was three days ago now and I can't reproduce it so I'll just take it as a dumb mistake on my part and close the issue. Maybe I mistyped --generate-mono-glue or something.

Thanks for all the help.
